dashboard/modules/metrics/export/grafana/dashboards/default_grafana_dashboard.json

```diff
@@ -87,7 +87,7 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "sum(ray_tasks) by (State)",
+          "expr": "sum(max_over_time(ray_tasks{State=~\"FINISHED|FAILED\"}[14d])) by (State) or clamp_min(sum(ray_tasks{State!~\"FINISHED|FAILED\"}) by (State), 0)",
```

Comment:
is this relevant to this PR?

Reply:
yes, we need to update the grafana task graph to account for the fact that we lose data after a worker exits.